src/main/scala/io/iohk/ethereum/crypto/ConcatKDFBytesGenerator.scala

```diff
@@ -16,7 +16,7 @@ class ConcatKDFBytesGenerator(digest: Digest) {
     * @param outputLength length of output that will be produced by this method,
     *                     maximum value is (digest output size in bits) * (2^32 - 1) but it should not be a problem
     *                     because we are using Int
-    * @throws IllegalArgumentException ("Output length too large") when outputLength is greater than (digest output size in bits) * (2^32 - 1)
+    * @throws scala.IllegalArgumentException ("Output length too large") when outputLength is greater than (digest output size in bits) * (2^32 - 1)
```

Comment:
Is there a reason to alter these? In general, formatting should be left as is, it generates 'noise' in the PR.